e2e/runner/sample-apps-shared/constants/sample-app-setup-configs.js

```diff
@@ -0,0 +1,38 @@
+const BRANCH_NAME = "release-x.53.x";
```

Comment:
This line affects only local testing. On CI the proper branch is calculated based on the release branch name

Reply:
probably good to add a comment or rename the constant to indicate this as it isn't immediately obvious